packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -481,6 +485,23 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
         }
     }, [useKeyboard, kioskOriginLocationId, mapsindoorsSDKAvailable]);
 
+    /*
+     * React on changes to the solution.
+     * Decide whether to show the Switch component depending on the modules enabled on the solution.
+     */
+    useEffect(() => {
```

Comment:
Assuming this is related to Mapbox only, and not Google Maps, would it make sense to move all this into the MapboxMap component? Also the use of the `<Switch />` component?

Reply:
Good idea, moved it to MapboxMap component ✅